src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -226,7 +238,20 @@ def _search(
                 query_kwargs.update(extra_query_kwargs)
 
             if query_kwargs["sort_by"] == "betterPriority":
-                query_kwargs["aggregate_kwargs"] = self.build_better_priority_sort_kwargs(request)
+                if features.has(
+                    "organizations:better-priority-sort-experiment",
```

Comment:
@ceorourke do we need to check `issue-list-better-priority-sort` to see if we should force to variant 1?